prow/flagutil/kubernetes_cluster_clients.go

```diff
@@ -391,7 +396,7 @@ func (o *KubernetesOptions) BuildClusterManagers(dryRun bool, requiredTestPodVer
 
 			// Check to see if we are able to perform actions against pods in
 			// the build cluster. The actions are given in requiredTestPodVerbs.
-			if err := checkAuthorizations(config, options.Namespace, requiredTestPodVerbs); err != nil {
+			if err := checkAuthorizations(config, "default", requiredTestPodVerbs); err != nil {
```

Comment:
The old field is gone in 0.15

https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.15.0

This is a basic check function so passing the default namespace should be fine.

Reply:
No, this is not fine. The permissions are usually granted for just the build pod namespace and not all namespaces.

Reply:
What should I put in there?

Reply:
Also, this is turning out to be alot more complicated than I though :(